src/w2form.js

```diff
@@ -1032,8 +1032,6 @@
                     case 'email':
                     case 'pass':
                     case 'password':
-                        field.el.value = value;
-                        break;
```

Comment:
Please take a look at this: http://plnkr.co/edit/tpl:heAMIR7Yae9GJCXW70nP?p=preview